appveyor.yml

```diff
@@ -27,9 +27,10 @@ build_script:
   - gn gen out/Default "--args=import(\"//electron/build/args/%GN_CONFIG%.gn\") %GN_EXTRA_ARGS%"
   - ninja -C out/Default electron:electron_app
   - gn gen out/ffmpeg "--args=import(\"//electron/build/args/ffmpeg.gn\") %GN_EXTRA_ARGS%"
-  - ninja -C out/ffmpeg third_party/ffmpeg
+  - ninja -C out/ffmpeg electron:electron_ffmpeg_zip
   - ninja -C out/Default electron:electron_dist_zip
   - appveyor PushArtifact out/Default/dist.zip
+  - appveyor PushArtifact out/Default/ffmpeg.zip
```

Comment:
not 100% sure whether this should target `out/Default/ffmpeg.zip` or `out/ffmpeg/ffmpeg.zip` since `gn gen` earlier builds to `out/ffmpeg`

Reply:
It is going to be in out/ffmpeg/ffmpeg.zip because that is where `ninja -C out/ffmpeg electron:electron_ffmpeg_zip` is going to put it.